### PR TITLE
Feat/high light card

### PR DESCRIPTION
**Pull request by Hudayberdiev2000**

### CODE REVIEW

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
Could you use .svg file in next Image instead of creating Icon component

File: src/components/atoms/HighlightCard/HighlightCard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import HighlightCard from "./HighlightCard";
+import { HighlightIcon } from "./HighlightIcon";
+
+export const Default = {
+  args: {
+    image: HighlightIcon,
+    title: "100 % Fresh Foods",
+    description:
+      "Beguiled and demoralized by all get charms pleasure the moments ever so blinded by desire.",
+    link: "https://tmcars.info/",
```
Comment (bruno-keiko) [inline src/components/atoms/HighlightCard/HighlightCard.stories.tsx]: you can replace the link argument with "/"

File: src/components/atoms/HighlightCard/HighlightCard.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      <figure className={styles["highlight-card__image-box"]}>{image}</figure>
+
+      <h3 className={styles["highlight-card__title"]}>{title}</h3>
+
+      <p className={styles["highlight-card__description"]}>{description}</p>
+
+      <Link
+        className={styles["highlight-card__link"]}
+        href={link}
+        target="_blank"
```
Comment (bruno-keiko) [inline src/components/atoms/HighlightCard/HighlightCard.tsx]: We don't need taget='_blank"

File: src/components/atoms/HighlightCard/HighlightCard.module.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+    margin-top: map.get($spacing, "md");
+    position: relative;
+    z-index: 1;
+
+    @include margin-bottom(xl);
+
+    &::after {
+      background-color: $highlight-card-color;
+      border-radius: 9999px;
```
Comment (bruno-keiko) [inline src/components/atoms/HighlightCard/HighlightCard.module.scss]: You can use br-full variable here

File: src/components/atoms/HighlightCard/HighlightCard.module.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  &__link {
+    border-radius: $br-sm;
+    bottom: map.get($spacing, md);
+    font-size: $fs-xl;
+    position: absolute;
+    right: map.get($spacing, xl);
+    text-decoration: none;
+    transition: all 0.3s; /* stylelint-disable-line unit-allowed-list */
```
Comment (bruno-keiko) [inline src/components/atoms/HighlightCard/HighlightCard.module.scss]: Please don't disable any rules.

**Comment by Hudayberdiev2000** [comment]
i made the change

**Comment by bruno-keiko** [comment]
> i made the change

Did you forget to push your changes

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
We need some changes

**Comment by Hudayberdiev2000** [comment]
Has made the changes

File: .stylelintrc.json [hunk 5]
```diff
@@ -6,7 +6,7 @@
     "declaration-property-unit-allowed-list": {
       "/^border/": ["px"]
     },
-    "unit-allowed-list": ["%", "deg", "px", "rem", "ms"],
+    "unit-allowed-list": ["%", "deg", "px", "rem", "ms", "s"],
```
Comment (bruno-keiko) [inline .stylelintrc.json]: Don't change rules.

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
Please don't change any rules

File: src/foundations/sass/_colors.scss [hunk 6]
```diff
@@ -24,15 +24,16 @@ $btn-primary-bg: var(--btn-primary-bg, $primary-color) !default;
...
 
 /* =============================================
-=                 DeliveryOrderCard                      =
+=                 DeliveryOrderCard            =
 ============================================= */
 
-$delivery-order-card-color: var(--delivery-order-card-color,
-    $secondary-color) !default;
+$delivery-order-card-color: var(
+  --delivery-order-card-color,
```
Comment (bruno-keiko) [inline src/foundations/sass/_colors.scss]: Please revert unnecessary style changes. You can use `npm run format:fix` instead of your IDE formatter.

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
Please revert to unnecessary style changes.

**Comment by Hudayberdiev2000** [comment]
resolved the issues

File: src/components/atoms/HighlightCard/HighlightCard.module.scss [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  &__link {
+    border-radius: $br-sm;
+    bottom: map.get($spacing, md);
+    font-size: $fs-xl;
+    position: absolute;
+    right: map.get($spacing, xl);
+    text-decoration: none;
+
+    // transition: all 0.3s;
+    transition: all #{0.3}s;
```
Comment (bruno-keiko) [inline src/components/atoms/HighlightCard/HighlightCard.module.scss]: Please change it to 300ms

**Review by bruno-keiko — CHANGES_REQUESTED** [review]

**Comment by Hudayberdiev2000** [comment]
I've changed "0.3s" to "300ms"

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
Could you resolve conflicts by rebasing?

**Comment by Hudayberdiev2000** [comment]
I have rebased..

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
We need to add `primary` and `secondary` modes. Could you look at the design?

**Comment by Hudayberdiev2000** [comment]
Did as you asked!

File: src/components/atoms/HighlightCard/HighlightCard.tsx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+  link: string;
+  mode: "primary" | "secondary";
+}
+
+const HighlightCard: React.FC<HighlightCardProps> = ({
+  image,
+  title,
+  description,
+  link,
+  mode = "secondary",
```
Comment (bruno-keiko) [inline src/components/atoms/HighlightCard/HighlightCard.tsx]: Please always use "primary" as the default value for clarity.

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
We need minor changes

**Comment by Hudayberdiev2000** [comment]
fixed, sorry for silly mistakes:)

**Review by bruno-keiko — APPROVED** [review]

**Pull request merged**